subprojects/core/src/integTest/groovy/org/gradle/api/PluginApplicationErrorIntegrationTest.groovy

```diff
@@ -41,7 +41,7 @@ apply plugin: 'broken'
 
         then:
         failure.assertHasCause("Failed to apply plugin [id 'broken']")
-        failure.assertHasCause("throwing plugin")
+        failure.assertHasCause("java.lang.Exception: throwing plugin")
```

Comment:
Yes. The root cause is now wrapped by an `UncheckedException` that change the message.